golem-worker-executor-base/src/durable_host/replay_state.rs

```diff
@@ -262,6 +264,7 @@ impl ReplayState {
                             function_name.to_string(),
                             request,
                             idempotency_key.clone(),
+                            InvocationContextStack::fresh(), // TODO: read from oplog
```

Comment:
is this TODO to do ?

Reply:
or not covered in this PR?

Reply:
It is coming in a next one. See the notion page with the roadmap.